lib/creator/index.js

```diff
@@ -15,18 +16,30 @@ const runTransform = require('./transformations/index');
 */
 
 function creator(options) {
-	const env = yeoman.createEnv(null, null, new WebpackAdapter());
-	const generatorName = options ? replaceGeneratorName(path.basename(options)) : 'webpack-default-generator';
+	let env = yeoman.createEnv('webpack', null, new WebpackAdapter());
+	const generatorName = options ? replaceGeneratorName(path.basename(options[0])) : 'webpack-default-generator';
 	if(options) {
-		const generatorName = replaceGeneratorName(path.basename(options));
-		env.register(require.resolve(options), generatorName);
+		const WebpackGenerator = class extends Generator {
+			initializing() {
+				options.forEach( (path) => {
+					return this.composeWith(require.resolve(path));
+				});
+			}
+		};
+		env.registerStub(WebpackGenerator, generatorName);
 	} else {
 		env.registerStub(defaultGenerator, 'webpack-default-generator');
 	}
 
 	env.run(generatorName)
 	.on('end', () => {
-		return runTransform(env.getArgument('configuration'));
+		if(generatorName !== 'webpack-default-generator') {
+			//HACK / FIXME
+			env = env.options.env;
```

Comment:
> Going to close this, as we're going to transition into using yeoman generators explicitly pretty soon. For now, running against a local fork is fine, but I'm a bit nervous about how the this.composeWith will turn out instead of env.configuration, when we run the webpack-cli generator in near future.

Based on this comment from Even, I think transitioning to explicit generator usage was the right call here. Let's file bugs upstream if there's more we can do to make this type of transition easier.